### PR TITLE
HTTP alert for Share Schedule popover

### PR DESCRIPTION
**Pull request by ezequielc**
## Description
UI style and HTTP error message conditionally shown if the user is visiting schedule details or not.

## Motivation and Context
Why is this change required? What problem does it solve?

## How Has This Been Tested?
Tested locally. Staged on [stage-1]

## Release Plan:
- As the Submitter, upon requesting review of this pull request, I confirm that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed. 
- As the Reviewer, upon approving the changes in this PR, I confirm I have reviewed and I agree that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed

#### Release Checklist Items Skipped?
If any Release Checklist items were intentionally skipped, please provide which ones and the reasons why

### CODE REVIEW

**Comment by ezequielc** [comment]
@alex-deaconu This is the UI and initial implementation of HTTP alert for Share Schedule. It is pending the actual check if HTTP items are present, as I assume you are implementing it as part of your current card. Thanks!

**Comment by alex-deaconu** [comment]
@ezequielc that is correct, see my branch/PR here: https://github.com/Rise-Vision/rise-vision-apps/pull/2029. Thanks.

**Comment by alex-deaconu** [comment]
@olegrise please review. Thanks!

File: test/unit/schedules/controllers/ctr-shared-schedule-popover.tests.js [hunk 1]
```diff
@@ -58,6 +74,26 @@ describe('controller: SharedSchedulePopoverController', function() {
     expect($scope.getLink()).to.equal('https://preview.risevision.com/?type=sharedschedule&id=scheduleId');
   });
 
+  describe('isScheduleDetails:', function() {
+    it('should should set isScheduleDetails to false if not in schedule details page', function() {
```
Comment (olegrise) [inline test/unit/schedules/controllers/ctr-shared-schedule-popover.tests.js:78]: @alex-deaconu `should should` :)

File: test/unit/schedules/controllers/ctr-shared-schedule-popover.tests.js [hunk 2]
```diff
@@ -58,6 +74,26 @@ describe('controller: SharedSchedulePopoverController', function() {
...
+      $state.current.name = 'apps.schedules.list';
+      $controller('SharedSchedulePopoverController', {
+        $scope: $scope
+      });
+      $scope.$digest();
+      expect($scope.isScheduleDetails).to.be.false;
+    });
+
+    it('should should set isScheduleDetails to true if visiting schedule details page', function() {
```
Comment (olegrise) [inline test/unit/schedules/controllers/ctr-shared-schedule-popover.tests.js:87]: @alex-deaconu `should should` :)

**Review by olegrise — APPROVED** [review]

File: test/unit/schedules/controllers/ctr-shared-schedule-popover.tests.js (hunk 2 above)
Comment (alex-deaconu) [reply test/unit/schedules/controllers/ctr-shared-schedule-popover.tests.js:87]: @olegrise good catch, I'll fix in the other branch. Thanks.

**Pull request merged**